tools/bro-to-vast/bro-to-vast.cpp

```diff
@@ -277,6 +277,8 @@ int main(int argc, char** argv) {
   vast::detail::add_message_types(cfg);
   vast::detail::add_error_categories(cfg);
   cfg.parse(argc, argv);
+  if (caf::get_or(cfg, "help", false))
+    return 0;
```

Comment:
@Neverlord The help-text I get from is currently:

```
global options:
  (-h|-?|--help)          : print help and exit
  --long-help             : print all help options and exit
  --dump-config           : print configuration in INI format and exit
  (-a|--address) <string> : the address where the endpoints listens
  (-p|--port) <uint16>    : the port where the endpoint binds to
  (-s|--show-progress)    : print one '.' for each proccessed event
```

I think it's confusing that `--long-help` and `--dump-config` are in there as well. If the user supplies either of these options, the program still executes, because I'm only checking for `help`.

Should I also add checks for `long-help` and `dump-config`?

Reply:
You can simply check `cfg.cli_helptext_printed` instead. This will catch all three cases.

Reply:
Good to know. I've tweaked the conditional accordingly.